draft-ietf-mls-protocol.md

```diff
@@ -75,6 +75,11 @@ informative:
        - name: Trevor Perrin(ed)
        - name: Moxie Marlinspike
 
+  SECG:
```

Comment:
Okay I see that we are referring to the uncompressed key format in SECG. I know there's a reference for that somewhere else, but I can't think of it right now. Go ahead and don't block on me and I can submit reference updates later.